PAWSModules/Package.swift

```diff
@@ -73,6 +77,27 @@ let package = Package(
         .target(
             name: "PAWSSharedContext",
             dependencies: []
+        ),
+        .target(
+            name: "PAWSLandingScreen",
+            dependencies: [],
+            resources: [
+                .process("Resources")
+            ]
```

Comment:
You can omit the dependencies list and the resources definition if a target does not use them or has no resources to resolve the warnings in your project:
```suggestion
            name: "PAWSLandingScreen"
```